embassy-stm32/src/rcc/h7.rs

```diff
@@ -614,6 +639,16 @@ pub(crate) unsafe fn init(mut config: Config) {
     // Peripheral Clock (per_ck)
     RCC.d1ccipr().modify(|w| w.set_ckpersel(ckpersel));
 
+    // ADC clock MUX
+    RCC.d3ccipr()
```

Comment:
we'll eventually need some crate-wide infra to manage all the stuf fin CCIPRx :S